### PR TITLE
Dac6 379: middle of arrangement journey

### PR DESCRIPTION
**Pull request by ers2015112**

### CODE REVIEW

File: app/utils/CheckYourAnswersHelper.scala [hunk 1]
```diff
@@ -36,6 +37,66 @@ import utils.CheckYourAnswersHelper.dateFormatter
 
 class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) {
 
+  def giveDetailsOfThisArrangement: Option[Row] = userAnswers.get(GiveDetailsOfThisArrangementPage) map {
```
Comment (iservport) [inline app/utils/CheckYourAnswersHelper.scala:40]: As a suggestion, the newly introduced rows could be separated in a trait and mixed up here.

**Review by iservport — APPROVED** [review]
LGTM

File: app/utils/CheckYourAnswersHelper.scala (hunk 1 above)
Comment (ers2015112) [reply app/utils/CheckYourAnswersHelper.scala:40]: This is just from the template though we could change it as long as the whole dev team is on the same page

**Comment by iservport** [comment]
LGTM

**Pull request merged**